client/state/selectors/get-media.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Returns media for a specified site ID and query.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @param  {Object}  query  Query object
+ * @return {?Array}         Media
+ */
+export default function getMedia( state, siteId, query ) {
+	const queries = state.media.queries[ siteId ];
+
+	if ( ! queries ) {
+		return null;
+	}
+
+	return queries.getItems( query );
```

Comment:
> Isn't this what the query managers are for?

What is "this"

What is "are for?"

It may be that the move into Redux demands some changes to the way we think about these things. In fact, if the query manager is working on its own data, a trivial change might be writing a fully-independent `getItems()`-type function which accepts the actual object being operated on as an input argument.

Keeping functions out of the state is pretty high up on the list of priorities for state design.